Source/Core/BoundingSphere.js

```diff
@@ -1302,5 +1304,14 @@ define([
         return BoundingSphere.clone(this, result);
     };
 
+    /**
+     * Computes the radius of the BoundingSphere.
+     * @returns {Number} The radius of the BoundingSphere.
+     */
+    BoundingSphere.prototype.volume = function() {
+        var radius = this.radius;
+        return (4.0 / 3.0) * CesiumMath.PI * radius * radius * radius;
```

Comment:
Assign `(4.0 / 3.0) * CesiumMath.PI` to a file-scoped constant so that it is not recomputed each time this function is called.  I'm not 100% sure the JIT will optimize it.